_posts/2019-11-05-wdc-webengine.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Web Data Connectors use QT WebEngine browser starting in Tableau 2019.4"
+abstract: "In Tableau 2019.4 and later, Web Data Connectors use an internal browser called QT WebEngine. This change offers WDC developers and users the features and capabilities available to modern browsers."
+---
+
+
+Prior to Tableau 2019.4, Web Data Connectors used an internal browser in Tableau called Qt WebKit. In Tableau 2019.4 and later, Web Data Connectors use Qt WebEngine (a Chromium-based browser). The Qt WebEngine is a significant upgrade over the previous browser and provides better support for HTML5 and modern JavaScript tools.
+
+Because of the change, users and developers of some WDCs might encounter issues. For more information, see [Troubleshoot WDC WebEngine Issues in Tableau 2019.4 and later]({{site.baseurl}}/docs/wdc_webengine.html).
+
+In addition, the debugging process for WDCs has changed. Developers will now be able to use the [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools){:target="_blank"}{:ref="noopener"} for debugging. For  more information, see the [Debugging a WDC in Tableau Desktop 2019.4 (and later)]({{site.baseurl}}/docs/wdc_debugging.html#debug-chrome).
```

Comment:
this target=blank looks strange on my local preview. Just making sure this does what it's supposed to do.

Reply:
Yes. It's kind of weird. The GitHub UI preview pane doesn't know what to do with the option. In the HTML page generated for the GitHub Pages from the Markdown, the `target="_blank"` doesn't  show up, but it opens the link in a new tab.